examples/production/values.yaml

```diff
@@ -46,6 +46,11 @@ spiffe-csi-driver:
   namespaceOverride: spire-system
   priorityClassName: system-node-critical
 
+
```

Comment:
because the server needs to talk to the agent associated with the upstream spire server. since its in a separate pod, it needs its own csi driver. Its needed for security reasons.